src/momento/simple_cache_client.py

```diff
@@ -55,26 +85,26 @@ def __exit__(
         self._control_client.close()
         self._data_client.close()
 
-    def create_cache(self, cache_name: str) -> CreateCacheResponse:
+    def create_cache(self, cache_name: str) -> CreateCacheResponseBase:
         """Creates a new cache in your Momento account.
 
         Args:
             cache_name: String used to create cache.
 
         Returns:
-            CreateCacheResponse
+            CreateCacheResponseBase
 
         Raises:
-            InvalidArgumentError: If provided cache_name is None.
+            InvalidArgumentError: If provided cache_name None.
```

Comment:
Super picky, but can you put the `is` back please? It just reads weird to me without it.

Also, not a blocker for this PR, but this `rasies` list is incomplete now. We should either fill it out with the other potential exceptions or maybe roll it up to `SdkException`.

Reply:
Thank you. I still need to update the docstrings. Will do that shortly.

Reply:
Updated documentation in https://github.com/momentohq/client-sdk-python/pull/179/commits/b5c4273620d84e8a3ae878c2208b231edde0bade, https://github.com/momentohq/client-sdk-python/pull/179/commits/0774d828c50c37727a6eec1f1cbc9d0b26c0cb7c, and https://github.com/momentohq/client-sdk-python/pull/179/commits/f7af5bede63be9c0ef0c916dd946921bbb745c8d.

Prior to this I used markdown code blocks (triple backtick, multiple lines of code, triple backtick) to render examples in the docstrings. I have found that this a VSCode feature and not an IntelliJ feature. The latter renders these blocks as garbage. Because of this I have backed out the markdown blocks in favor in just plain indented blocks. This matches the Google docstring guide.